packages/shared_preferences/shared_preferences_android/CHANGELOG.md

```diff
@@ -1,38 +1,39 @@
-## NEXT
+## 2.0.16
 
-* Updates minimum Flutter version to 3.0.
+- Updates minimum Flutter version to 3.0.
```

Comment:
Seems like your linter may have made some unwanted changes to this file

Reply:
Really, sorry I didn't see that. To confirm we are on the same page, you talking about the `*` changing to `-`  right?